src/styles/_media.scss

```diff
@@ -0,0 +1,169 @@
+:root {
+    --background-image: transparent;
+    --background-overlay: transparent;
+}
+
+.media--container {
+   margin-top: 100px;
+}
+
+.overview--container {
+    background: var(--background-image);
+    background-repeat: no-repeat;
+    background-size: 87% 100%;
+    background-position: right;
+}
+
+.overlay--bg {
+    background-image: var(--background-overlay);
+    @include flexContainer(row, start, start);
+    padding: 30px 30px;
+    gap: 45px;
+}
+
+.poster--container {
+    @include flexContainer(column, center, none);
+}
+
+.poster--media {
+    border-radius: 10px 10px 0px 0px;
+    box-shadow: 0px 20px 41px #000;
+}
+
+.effect--front {
+    border: none;
+    margin: 0 auto;
+}
+
+.effect--front:hover {
+    -webkit-transform: scale(1.1);
+    -moz-transform: scale(1.1);
+    -o-transform: scale(1.1);
+    transform: scale(1.1);
+    transition: all 0.3s;
+    -webkit-transition: all 0.3s;
+}
+
+
+.container--provider {
+    @include flexContainer(row, center, center);
+    gap: 12px;
+    background-color: $background;
+    color: $text-color2;
+    padding: 10px 0;
+    border-radius: 0 0 10px 10px;
+
+    & a {
```

Comment:
**ALERT** please read what the `&` operator does in SASS. What you are doing here is the same as:
```suggestion
    a {
```

Same for the following cases. You use `&` to repeat the parent class.